src/pkg/scan/report/summary.go

```diff
@@ -85,7 +86,9 @@ func MergeNativeSummary(s1, s2 interface{}) (interface{}, error) {
 
 // SupportedGenerators declares mappings between mime type and summary generator func.
 var SupportedGenerators = map[string]SummaryGenerator{
-	v1.MimeTypeNativeReport: GenerateNativeSummary,
+	v1.MimeTypeNativeReport:               GenerateNativeSummary,
+	v1.MimeTypeGenericVulnerabilityReport: GenerateNativeSummary,
+
```

Comment:
remove this empty line to fix the gofmt lint issue

Reply:
Done